frontend/src/app/modules/home/home-page/home-page.component.ts

```diff
@@ -104,6 +109,12 @@ export class HomeComponent extends BaseComponent implements OnInit, OnDestroy {
             });
     }
 
+    async changeOrganization(organization: Organization) {
+        this.authService.setOrganization(organization);
+        await this.router.navigateByUrl('home/organization/settings');
+        window.location.reload();
```

Comment:
Why do we need this line?

Reply:
This need to reload the page to get new data

Reply:
Please don't use reloading the page becouse it is SPA.

Reply:
Ok